smb_slam/launch/localization.launch

```diff
@@ -5,6 +5,7 @@
   <arg name="launch_sensors"      default="false"/>
   <arg name="use_lidar_odometry"  default="false"/>
   <arg name="map_name"            default="Wangen_mapdecimated.pcd" /> 
+  <arg name="pcd_filepath"        default="$(find smb)/data/$(arg map_name)" />
```

Comment:
Is there a particular reason why you would store the pcd in the `smb` package instead in the `smb_slam` by default? By default, there is no folder `data` in the smb package.

Reply:
good point, not sure why I did that so I'll change this to smb_slam